app/models/manageiq/providers/openstack/cloud_manager/cloud_network/public.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::Openstack::CloudManager::CloudNetwork::Public < ManageIQ::Providers::Openstack::CloudManager::CloudNetwork
```

Comment:
@matthewd so CloudNetwork::Public, or CloudNetworkPublic or PublicCloudNetwork

I just realized PublicCloudNetwork will be easier, so I can just write has_many :puiblic_cloud_networks without specifying class

So I guess you already have idea how we should do that. :-)